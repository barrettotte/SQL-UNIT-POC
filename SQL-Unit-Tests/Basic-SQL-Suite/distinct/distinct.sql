-- This is a simple test for a DISTINCT statement --

INSERT INTO [BARRETT_TEST].[dbo].[roles] (role_name) VALUES ('TEST');
INSERT INTO [BARRETT_TEST].[dbo].[roles] (role_name) VALUES ('WASD');
INSERT INTO [BARRETT_TEST].[dbo].[roles] (role_name) VALUES ('QWERTY');
INSERT INTO [BARRETT_TEST].[dbo].[roles] (role_name) VALUES ('TEST');
INSERT INTO [BARRETT_TEST].[dbo].[roles] (role_name) VALUES ('TEST');

SELECT DISTINCT role_name FROM [BARRETT_TEST].[dbo].[roles];
