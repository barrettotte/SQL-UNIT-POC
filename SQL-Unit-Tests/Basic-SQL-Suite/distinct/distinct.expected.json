{
    "success": "true",
    "columns": [
        {
            "name": "role_name",
            "type": "VARCHAR(50)"
        }
    ],
    "rows": [
        {
            "role_name": "QWERTY"
        },
        {
            "role_name": "ROLE_ADMIN"
        },
        {
            "role_name": "ROLE_DEVELOPER"
        },
        {
            "role_name": "ROLE_USER"
        },
        {
            "role_name": "TEST"
        },
        {
            "role_name": "WASD"
        }
    ]
}